pkg/daemon/ceph/config/info.go

```diff
@@ -47,7 +48,7 @@ type MonInfo struct {
 // of the ClusterInfo struct during startup, specifically that it is expected to exist after the
 // Rook operator has started up or connected to the first components of the Ceph cluster.
 func (c *ClusterInfo) IsInitialized() bool {
-	if c == nil || c.FSID == "" || c.MonitorSecret == "" || c.AdminSecret == "" {
+	if c == nil || c.FSID == "" || c.MonitorSecret == "" || c.AdminSecret == "" || c.CSISecret == "" {
```

Comment:
@travisn @BlaineEXE I'm expecting this to break upgrade so we need a solution.

Reply:
I'm thinking of adding a procedure in the upgrade guide from 1.1 to 1.2 which will create the secret in ceph as well as adding it in the configmap.

Reply:
Has the upgrade issue been solved at this point?

Reply:
Yes.